Firestore/src/FirestoreClient.php

```diff
@@ -207,6 +207,10 @@ public function batch()
      *     @type int $maxBatchSize Maximum number of requests per batch.
      *     @type bool $greedilySend Flag to indicate whether BulkWriter greedily
      *           sends batches.
+     *     @type bool $isThrottlingEnabled Flag to indicate whether rate of
+     *           sending writes can be throttled.
+     *     @type int $initialOpsPerSecond Initial number of operations per second.
+     *     @type int $maxOpsPerSecond Maximum number of operations per second.
```

Comment:
WDYT about documenting the default values here?

Reply:
Added